CPT331.WebAPI/Api/TestController.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Web.Http;
+
+namespace CPT331.WebAPI.Api
+{
+    public class TestController : ApiController
+    {
+        // GET api/<controller>
+        public IEnumerable<string> Get()
+        {
+            return new string[] { "value1", "value2" };
+        }
+
+        // GET api/<controller>/5
+        public string Get(int id)
+        {
+            return "value1";
+        }
+
+        // POST api/<controller>
+        public void Post([FromBody]string value)
```

Comment:
This Controler will actually be **/api/Test** as it uses the deafult API route (defined in the App_Start code).

This will support two types of get requests:
**Get Complete List as IEnumerable**
/api/Test 

**Get specific item using its id**
/api/Test/{id}
e.g.
/api/Test/1

The id variable is defined in the API default route.


If we need further routes, we can move away from default routes by using the Route Attribute (although it makes the API a little messy).  
[Route("api/something/something/{variable1}/{variable2}")]